kpops/components/base_components/kafka_app.py

```diff
@@ -39,6 +42,11 @@ class KafkaApp(KubernetesApp):
 
     _type = "kafka-app"
     app: KafkaAppConfig
+    repo_config = HelmRepoConfig(
+        repository_name="bakdata-streams-bootstrap",
+        url="https://bakdata.github.io/streams-bootstrap/",
+    )
+    version = "2.7.0"
```

Comment:
isn't this specific to streams-bootstrap only?

Reply:
Streams-bootstrap is a KafkaApp, which is applied into : ProducerApp and StreamsApp. They share both the same helm chart repository

Reply:
ok, if the `KafkaApp` class IS streams-bootstrap specific I think we should update the docstring
> Base component for kafka-based components.

because this sounds rather generic

Reply:
Well our understanding is that every KafkaApp uses StreamsBootstrap